frontend/vite.config.ts

```diff
@@ -24,6 +25,11 @@ const config = ({ mode }: ConfigEnv): ReturnType<typeof defineConfig> => {
         },
       },
     },
+    resolve: {
+      alias: {
+        '@img': path.resolve(__dirname, './src/assets/img'),
```

Comment:
Maybe it would be better to link assets folder, in case if we'd need to get path for anything else
```suggestion
        '@assets': path.resolve(__dirname, './src/assets'),
```